### ISSUE TITLE
Spindle flyback diode

### ISSUE THREAD

**Issue by m1ngr** [secures92/CyclneGRBLCtrl#3]
Hi, are you sure about the D2 Spindle diode? Seems to be reversed.

**Comment by secures92** [comment]
You are right. Thanks. Its reversed. I've realized it after assembling. I did never commit the corrected schematic :S. Sry
